<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>holiday</class>
 <widget class="QMainWindow" name="holiday">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1370</width>
    <height>985</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>holiday</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>431</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>Kunden</string>
    </property>
    <widget class="QTableWidget" name="ReisenBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>391</width>
       <height>241</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Reise ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Beginn der Reise</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ende der Reise</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
    </widget>
    <widget class="QTextEdit" name="KundenId">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>70</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="Kundenname">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Reisen</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>581</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>Reisen</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>70</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Reise ID</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="ReiseID">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Buchungen</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="BuchungenBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>511</width>
       <height>231</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Buchungstyp</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Start</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ende der Reise</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Preis</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>20</y>
      <width>611</width>
      <height>781</height>
     </rect>
    </property>
    <property name="title">
     <string>Buchungdetails</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>BuchungId</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="BuchungID">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="Preis">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>160</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Preis</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Ende</string>
     </property>
    </widget>
    <widget class="QTabWidget" name="Tabs">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>230</y>
       <width>461</width>
       <height>411</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="Hotel">
      <attribute name="title">
       <string>Hotel</string>
      </attribute>
      <widget class="QTextEdit" name="Hotelname">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Stadt</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="Stadt">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>60</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>HotelNmae</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="Zimmer">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>110</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Zimmertyp</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="Zimmer_2">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>210</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="Zimmer_3">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>160</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Hotellongitude</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>HotelLatitude</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="Flight">
      <attribute name="title">
       <string>Flight</string>
      </attribute>
      <widget class="QTextEdit" name="FromDest">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>20</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>To</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="ToDest">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>60</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>From</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="Airline">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>110</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Airline</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="Vehicleclass_6">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>150</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_26">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>300</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>ToLongitude</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="Vehicleclass_7">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>300</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_27">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>250</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>ToLatitude</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_28">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>200</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Fromlongoitude</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_29">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>150</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>FromLatitude</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="Vehicleclass_8">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>200</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="Vehicleclass_9">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>250</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="RentalCar">
      <attribute name="title">
       <string>RentalCar</string>
      </attribute>
      <widget class="QTextEdit" name="Pickup">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>20</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>60</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Return</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="Return">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>60</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Pickup</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="Company">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>100</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>100</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Company</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>140</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Vehicleclass</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="Vehicleclass">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>140</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="Vehicleclass_2">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>340</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="Vehicleclass_3">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>190</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="Vehicleclass_4">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>240</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="Vehicleclass_5">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>290</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>330</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>ReturnLongitude</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_23">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>240</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Pickuplongoitude</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_24">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>190</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>PickupLatitude</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_25">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>290</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>ReturnLatitude</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QDateEdit" name="StartDate">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="Enddate">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>110</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>650</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>650</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>650</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Check Airport</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Createfile">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>650</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>CreatFile</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>720</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Sorting </string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>720</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Check Travels</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>720</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>ABC Analyse</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1370</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEinlesen">
    <property name="title">
     <string>Einlesen</string>
    </property>
    <addaction name="actionInformation_Anzieigen"/>
    <addaction name="actionShow_Airport"/>
    <addaction name="actionhello"/>
   </widget>
   <widget class="QMenu" name="menuBuchung">
    <property name="title">
     <string>Buchung</string>
    </property>
    <addaction name="actionAlle_Anzeigen"/>
    <addaction name="actionAnzeigen_nach_ID"/>
   </widget>
   <addaction name="menuEinlesen"/>
   <addaction name="menuBuchung"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAlle_Anzeigen">
   <property name="text">
    <string>New Customer</string>
   </property>
  </action>
  <action name="actionAnzeigen_nach_ID">
   <property name="text">
    <string>Anzeigen nach ID</string>
   </property>
  </action>
  <action name="actionInformation_Anzieigen">
   <property name="text">
    <string>Information Anzieigen</string>
   </property>
  </action>
  <action name="actionShow_Airport">
   <property name="text">
    <string>Show Airport</string>
   </property>
  </action>
  <action name="actionhello">
   <property name="text">
    <string>Sort</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
